fake-manage-recalls-api/stubs/__files/get-recall-with-sentencing-info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "recallId": "8ab377a6-4587-2598-abc4-98fc53737",
+  "nomsNumber": "A1234AA",
+  "sentenceDate": "2019-08-03",
+  "licenceExpiryDate": "2021-08-03",
+  "sentenceExpiryDate": "2021-02-03",
+  "sentencingCourt": "Manchester Crown Court",
+  "indexOffence": "Burglary",
+  "conditionalReleaseDate": "2021-12-03",
+  "sentenceLength": {
```

Comment:
this is already in the get-recall stub?

Reply:
(Aside!  ^^^ Perhaps an example for review in our "to be arranged" discussion of testing in this project? - per our Dev get-together of Wednesday)?

Reply:
I'm confused :)
I'm just saying there's no need to create a new stub JSON file, we already have - https://github.com/ministryofjustice/manage-recalls-ui/blob/main/fake-manage-recalls-api/stubs/__files/get-recall.json#L30-L34

Reply:
That depends on what exactly you want to test.  Our usage of that API at the moment would never return a fully populated Recall after saving sentencing info.  I discussed this briefly with Andy, and we need to agree our level of testing with Pact.

Reply:
@AndyNewton you mentioned 'maximal' and 'minimal' examples, which sounds right to me. If that's the case, there's no need for an additional stub, as the existing get-recall.json stub is the 'maximal' example.